.azure-pipelines/azure-pipelines-windows.yml

```diff
@@ -222,7 +222,7 @@ jobs:
       - script: |
           setlocal EnableDelayedExpansion
           set PATH=$(Build.BinariesDirectory)\install\bin;%PATH%
-          psi4 $(Build.SourcesDirectory)\tests\tu1-h2o-energy\input.dat
+          psi4 $(Build.SourcesDirectory)\tests\tu1-h2o-energy\input.dat -o stdout
```

Comment:
Are we wanting to keep `-o stdout`? I don't care either way.

Reply:
I kept that deliberately, so we get output when something goes wrong.